### DISCUSSION TITLE
Add products to a shop abroad

### DISCUSSION THREAD

**Discussion by verilyanton** [plante-app-team/plante#285] [Features]
Recently I've been on the road and shopping for vegan food in supermarkets. In some places I didn't have time to add the products to the shop right away, however after arriving to a different city, it was quite more difficult to locate it on the map, because the map wouldn't zoom out enough. I'm not really familiar with OSM api but I assume that limitation can be changed. @blazern what do you think?

**Comment by blazern** [comment]
The Zoom Out limit is not related to OSM, and unfortunately I don't remember the reason for the limitation.

The limits are set in `MapPageMode` and the classes that extend it by either `minZoom` or `maxZoom` method (I don't remember which).
You can play around with values locally to see how they could be better set, and then make a Pull Request.
